python/cuspatial/cuspatial/tests/test_geoseries.py

```diff
@@ -350,155 +352,55 @@ def test_size(gs, series_slice):
     assert len(gi) == len(cugs)
 
 
-def test_geometry_point_slicing(gs):
-    cugs = cuspatial.from_geopandas(gs)
-    assert (cugs[:1].points.x == cudf.Series([-1])).all()
-    assert (cugs[:1].points.y == cudf.Series([0])).all()
-    assert (cugs[:1].points.xy == cudf.Series([-1, 0])).all()
-    assert (cugs[3:].points.x == cudf.Series([9])).all()
-    assert (cugs[3:].points.y == cudf.Series([10])).all()
-    assert (cugs[3:].points.xy == cudf.Series([9, 10])).all()
-    assert (cugs[0:4].points.x == cudf.Series([-1, 9])).all()
-    assert (cugs[0:4].points.y == cudf.Series([0, 10])).all()
-    assert (cugs[0:4].points.xy == cudf.Series([-1, 0, 9, 10])).all()
-
-
 def test_geometry_multipoint_slicing(gs):
+    points_list = gs[gs.apply(lambda x: isinstance(x, (MultiPoint, Point)))]
+    points = list(
+        chain(points_list.apply(get_coordinates))
+    )  # flatten multilinestrings/linestrings
+    coords_list = list(chain(*points))  # flatten linestrings
+    xy_interleaved = list(chain(*coords_list))  # flatten coordinates
+    x = xy_interleaved[::2]
+    y = xy_interleaved[1::2]
+
     cugs = cuspatial.from_geopandas(gs)
-    assert (cugs[:2].multipoints.x == cudf.Series([1, 3])).all()
-    assert (cugs[:2].multipoints.y == cudf.Series([2, 4])).all()
-    assert (cugs[:2].multipoints.xy == cudf.Series([1, 2, 3, 4])).all()
-    assert (cugs[2:].multipoints.x == cudf.Series([5, 7])).all()
-    assert (cugs[2:].multipoints.y == cudf.Series([6, 8])).all()
-    assert (cugs[2:].multipoints.xy == cudf.Series([5, 6, 7, 8])).all()
-    assert (cugs[0:4].multipoints.x == cudf.Series([1, 3, 5, 7])).all()
-    assert (cugs[0:4].multipoints.y == cudf.Series([2, 4, 6, 8])).all()
-    assert (
-        cugs[0:4].multipoints.xy == cudf.Series([1, 2, 3, 4, 5, 6, 7, 8])
-    ).all()
+    assert (cugs.points.x == cudf.Series(x)).all()
+    assert (cugs.points.y == cudf.Series(y)).all()
+    assert (cugs.points.xy == cudf.Series(xy_interleaved)).all()
```

Comment:
One small note: I believe the original test case also tests the `__getitem__` operator of `cugs`. Although the way how the gs is setup actually makes no impact to whether the slicing is involved or not. (meaning `cugs[0:4]` contains only point / multipoint anyway). But this still tests more code paths then the current test. I'd suggest also include the slicing here.

```suggestion
    assert (cugs[0:4].points.x == cudf.Series(x)).all()
    assert (cugs[0:4].points.y == cudf.Series(y)).all()
    assert (cugs[0:4].points.xy == cudf.Series(xy_interleaved)).all()
```

Reply:
Re-added slicing. Thanks!